<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EwFirstPage</class>
 <widget class="QWizardPage" name="EwFirstPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="title">
   <string>Format &amp; options</string>
  </property>
  <property name="subTitle">
   <string>Choose the export format and path</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Path :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="pathLineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="browseButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="formatGroupBox">
     <property name="title">
      <string>Format</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pathLineEdit</tabstop>
  <tabstop>browseButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
